packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -413,9 +414,19 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 this.logger,
                 { eventName: "ObtainSnapshot" },
                 async (event: PerformanceEvent) => {
-                    let cachedSnapshot: ISnapshotContents | undefined;
-                    const cachedSnapshotP: Promise<ISnapshotContents | undefined> =
-                        this.epochTracker.get(createCacheSnapshotKey(this.odspResolvedUrl));
+                    const props = {};
+                    let cachedSnapshot: ISnapshotContents | ISnapshotCachedEntry | undefined;
+                    const cachedSnapshotP: Promise<ISnapshotCachedEntry | undefined> =
```

Comment:
Given that downstream everything works with ISnapshotContents, I think you can actually keep both types (lines 418 & 419) as ISnapshotContents.
ISnapshotCachedEntry was only useful if you were using it down the road.